demo-app/src/components/RenderFromEndpoint.tsx

```diff
@@ -35,23 +27,13 @@ export default function RenderFromEndpoint({
     setLoading(false)
   }, [url])
 
-  const configurationFunction = React.useMemo(
-    () => applyConfiguration(components),
-    [],
-  )
-
-  const renderedComponents = React.useMemo(() => {
-    if (config) {
-      const parsedConfig = parseYaml(config)
-      return configurationFunction(parsedConfig, {})
-    }
-  }, [config])
-
   if (error) {
     return <>OOPS! {error.message}</>
   } else if (loading) {
     return <>Loading...</>
+  } else if (config) {
+    return <RenderConfig config={config} />
   }
 
-  return <>{renderedComponents}</>
+  return <div>No config :(</div>
```

Comment:
Feels like the default return should be RenderConfig and the else if case above could be changed to `!config`.